stripe/api_resources/account.py

```diff
@@ -3,11 +3,11 @@
 import stripe
 from stripe import oauth, six, util
 from stripe.api_resources.abstract import CreateableAPIResource
+from stripe.api_resources.abstract import custom_method
```

Comment:
Nit: I think the sort order should be case-sensitive, i.e. `custom_method` should appear after `UpdateableAPIResource`.

No need to act specifically for this though, as I expect `custom_method` will disappear fairly soon and we'll just generate the methods directly instead.